demo/demo.sh

```diff
@@ -4,8 +4,10 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT="${THIS_DIR}/../"
 SERVER_DIR="${ROOT}/pipeline_server/"
 
+service postgresql restart
```

Comment:
Why do we need to restart Postgres?

Reply:
Postgres isn't started by default when running the container for the first time. We should probably should just do "service postgresql start" in the login script instead.